test/activation_based/test_ann2snn.py

```
@@ -953,14 +1526,57 @@ def test_mha_causal_attention_rejects_explicit_mask_at_runtime(self):
 
 
 class TestFuse:
+    def test_fuse_module_replacement_does_not_depend_on_asserts(self):
+        code = """
+import inspect
+from spikingjelly.activation_based.ann2snn import RateCodingRecipe
+
+source = inspect.getsource(RateCodingRecipe._fuse)
+raise SystemExit(int("assert isinstance(node.target, str)" in source))
+"""
+        result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)